### ISSUE TITLE
discover lists bogus directories as calendars

### ISSUE THREAD

**Issue by hobarrera** [pimutils/khal#375]
Wow, khal moved to `/pimutils` just as I posted the issue a moment ago, and the issue disappeared!

---

I've this config:
```
[calendars]
 [[calendars]]
  path = ~/.local/share/calendars/
  type = discover
```

Since `~/.local/share/calendars` is a git repository, khal lists all subdirectories in `~/.local/share/calendars/.git` as additional calendars - that's dozens of subdirectories inside ikhal's calendar picker.

Maybe dotfile-directories should be ignored when using discover?

**Comment by geier** [comment]
yeah, probably a good idea

**Comment by geier** [comment]
fixed with #425
